lib/sinon/behavior.js

```diff
@@ -156,7 +158,12 @@ var proto = {
             return (this.promiseLibrary || Promise).reject(this.returnValue);
         } else if (this.callsThrough) {
             return this.stub.wrappedMethod.apply(context, args);
+        } else if (this.returnValueDefined) {
+            return this.returnValue;
+        } else if (this.yields) {
```

Comment:
Is a new property really required? I think checking for `callsArgAt` would also do, no?

Reply:
It would seem that callArgAt will work for this.